app/tests/test_token_repository.py

```diff
@@ -0,0 +1,67 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+import boto3
+from docker import from_env, DockerClient
+from docker.models.containers import Container
+
+
+class TestTokenRepository(TestCase):
+    dynamodb_container: Container = None
+
+    @classmethod
+    def setUpClass(cls) -> None:
```

Comment:
It could be a good idea to isolate the setupClass and the Teardown in a dedicated `DynamodbTestCase` class that inherits TestCase to remove the noise due to the db setup and improve test readability.

We could then inherit this `DynamodbTestCase` for every integration repository test suites that needs it.

I would also put the _run_dynamodb_container function there

Reply:
Could you provide some PR for that? I am not sure to understand what you mean by 'for every integration repository test suites' as we plan to have only one repository at the moment.